deno/payloads/v10/auditLog.ts

```diff
@@ -517,6 +526,16 @@ export type APIAuditLogChangeKeyPermissions = AuditLogChangeData<'permissions',
  */
 export type APIAuditLogChangeKeyColor = AuditLogChangeData<'color', number>;
 
+/**
+ * Returned when application command permissions are changed
+ */
+export type APIAuditLogChangeKeyCommandId = AuditLogChangeData<'command_id', string>;
+
+/**
+ * Returned when permissions for a command were updated
+ */
+export type APIAuditLogChangeKeySnowflake = AuditLogChangeData<'snowflake', Snowflake>;
```

Comment:
These two are wrong, one of the possibilities is now the `change.key` can be a Snowflake and have a value that for all intents and purposes we could only strictly type as unknown... (for this one PR the value is APICommandPermissionData or w/e)